### ISSUE TITLE
Multiple install folder locations

### ISSUE THREAD

**Issue by lawlegge** [flybywiresim/installer#401]
### Installer Version

v3.2.2

### Description

Please allow to set individual install folders for the A32NX and the 74S in the installer. It's a major hassle to have to change the install folder locations back and forth in the installer...

**Comment by tracernz** [comment]
To help understand the use case... you are using the two add-ons in two different copies of the sim?

**Comment by lawlegge** [comment]
Hi Michael,
Thank you for the reply.

No, just 1 sim, but individual install locations for each package.
Basically I'm requesting the ability to specify/set individual install
locations for the individual content e.g.; A32NX to its install directory,
the 74S to its install directory, etc...

This pic illustrates what I'm looking for...
[image: 2022-09-23 install directory 1_4.jpg]

Here is my install configuration for the aircraft, the folder
tree...individual locations...
[image: 2022-09-23 tree1_1.jpg]

I hope this gives a better explanation of what I'm looking for.


Thank you folks for everything you do..it is GREATLY appreciated...


Best Regards,

Law

**Comment by FoxtrotSierra6829** [comment]
Sadly, we cannot see your image. Could you please upload it directly via GitHub?

**Comment by mcnv232** [comment]
After revisiting the FBW installer after some time, I've also seen this as an issue. It would be a very nice improvement for the installer to create directories, such as how Orbx can install files, especially with larger installs such as FSTL's base models (It would be incredible if the FBW installer could scan for directories created in other installers, if possible).

**Comment by Benjozork** [comment]
This is planned.

**Comment by St54Kevin** [comment]
I also want to request this Feature as we now have different types of addons except just planes. My use case is a organized folder structure by type to be used with AddonsLinker. Eg i would want to put the model matching tool into my tools folder.

**Comment by marchershey** [comment]
I second this. 

I would like to be able to set the install directory of each item separately because I'm using Addon Linker, and with very specific, organized folder structure (Aircraft, Scenery, Tools, etc), it makes it difficult to use your installer properly. I have to jump onto your installer, redownload the aircraft/tool then move the files over to the correct installation folder. A32NX is an aircraft, SimBridge is a tool. They are in separate folders on my system. Hopefully this makes sense.

**Comment by FoxtrotSierra6829** [comment]
Are categories aircraft, scenery, tools... enough or do you request individual selection on a per-addon basis?

**Comment by marchershey** [comment]
> Are categories aircraft, scenery, tools... enough or do you request individual selection on a per-addon basis?

Honestly, I think individual selections per addon would be best.

Because the installer offers two different types of aircraft. The A32NX, and the Salty 747. My folders are organized by aircraft type. 

Example:
`Aircraft/A320s/FBW-A32NX`
`Aircraft/747s/SALTY-B747`

So having just a single location for aircraft would prevent that. 

Hope that makes sense!

**Comment by trevorhannant** [comment]
I agree with this, even more so now that there is FSLTL in there.   I have the same idea on folder structure for use with AddOn Linker:

![image](https://github.com/user-attachments/assets/fb81efa3-d089-4207-a00b-dcac1f900808)

Being able to set specific paths per add-on within the Installer, especially if different types of add-on get integrated later would be advantageous.
